examples/data_apis/tensorflow_data_api_tiledb_sparse.ipynb

```diff
@@ -0,0 +1,553 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This example notebook shows how we can train a simple Regression classifier.\n",
+    "We employ TileDB as a storage engine for our training data and labels.\n",
+    "We will use the MovieLens 100K public data set, available [here](https://grouplens.org/datasets/movielens/100k/). We will first download the\n",
+    "MovieLens, which contains 100.000 ratings, by 943 users on 1682 items.\n",
+    "Continuing, we will use our TileDB support for Tensorflow Sparse Dataloader API in order to train the classifier.\n",
+    "First, let's import what we need and download our data. We will transform our data to a sparse format\n",
+    "in order to show the support of TileDB in ingesting and providing to the Tensorflow framework sparse datasets. Sparse\n",
+    "datasets are important and frequently found and used in applications like recommender systems et.al. For example by designing\n",
+    "a Factorisation Machine [FM model](https://www.csie.ntu.edu.tw/~b97053/paper/Rendle2010FM.pdf) someone could take advantage of data sparsity and build a refined recommendation system."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib.request\n",
+    "import numpy as np\n",
+    "import tiledb\n",
+    "import tensorflow as tf\n",
+    "import pandas as pd"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Dataset\n",
+    "Download MovieLens dataset."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "('u.data', <http.client.HTTPMessage at 0x10eb2cca0>)"
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "url = 'https://files.grouplens.org/datasets/movielens/ml-100k/u.data'\n",
+    "filename = 'u.data'\n",
+    "urllib.request.urlretrieve(url, filename)"
```

Comment:
This is failing for me with 
`URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)>`

Reply:
Maybe due to multi-line comment is not shown as resolved. However I changed the url to `http`